text.py

```diff
@@ -0,0 +1,30 @@
+from element import Element
+from size import Size
+from pos import Pos
+from font import font_surface
+
+import colour
+import pygame
+
+class Text(Element):
+    def __init__(self, size=Size(70, 30), text='', font_size=11):
+        super(Text, self).__init__(size)
+        self.text = text
+        self.font_size = font_size
+        self.border = 2
+
+    def render(self, surface):
```

Comment:
This shares a lot of logic with the `Button` class. Let's find a way to reduce duplicate code. Perhaps try subclassing.